apps/api/src/common/convert-activity-gap-table-to-csv.ts

```diff
@@ -0,0 +1,54 @@
+const { parse } = require('json2csv');
+
+interface CareActivityProps {
+  name: string;
+  'Respiratory Therapist': string;
+  'Speech Language Pathologist': string;
+  'Occupational Therapist': string;
+  'Registered Nurse - Critical Care': string;
+  'Registered Nurse - Medical/ Surgical': string;
+  'Licensed Practical Nurse (LPN)': string;
+  'Health Care Assistant (HCA)': string;
+  'Other - Non Clinical': string;
+  Physician: string;
+  numberOfGaps?: string;
+}
+interface JsonDataProps extends CareActivityProps {
+  careActivities: CareActivityProps[];
+}
+
+export const convertActivityGapTableToCSV = (data: any) => {
+  try {
+    const fields = data.headers.map((element: string | { label: string; value: string }) => {
+      if (element === 'Activities Bundle') return { label: 'Activities Bundle', value: 'name' };
+      return element;
+    });
+
+    const emptyRow: CareActivityProps = {
+      name: '',
+      'Respiratory Therapist': '',
```

Comment:
This occupation data should be constructed using the `headers` response object and shouldn't be hardcoded

Reply:
Used headers as suggested